Makefile

```diff
@@ -267,6 +267,12 @@ fmt: ## Format the code
       	else \
       		echo "Fixed formatting for: $(FORMATTED)"; \
       	fi
+JX_FILES="pkg/cmd/*.go"
+gettext:
+	go-xgettext -k=i18n.T "${JX_FILES}" > pkg/i18n/jx.pot
+
+gen-data:
+	cd pkg/i18n && go-bindata -o bindata.go -pkg i18n jx/zh_CN/LC_MESSAGES/
```

Comment:
So you are using go-bindata to include the translations into the binary, right? I used go-bindata in another project and we got big problems when it suddenly disappeared. Seems the owner brought at least the repo back, but it is archived. Not sure I like the idea of depending on go-bindata. Are there any alternatives?

Reply:
BTW, in terms of the `Makefile`, we would have to make sure that `go-bindata` as well as `go-xgettext` are getting installed.

Reply:
Yes, I met the `go-bindata` issue too. I given a temporary solution in [another project](https://github.com/jenkins-zh/jenkins-cli/blob/master/Makefile#L60). I don't have an alternative one for now.